pipenv/core.py

```diff
@@ -2473,8 +2471,8 @@ def do_run(project, command, args, three=None, python=False, pypi_mirror=None):
         project, three=three, python=python, validate=False, pypi_mirror=pypi_mirror,
     )
 
+    load_dot_env(project)
```

Comment:
I am afraid this will break the isolation in test cases. dotenv files in one test case will contaminate other cases env.

Reply:
@frostming Could you elaborate on this?  Right now the test suite is passing with this change so I am not sure I understand, but maybe I do.  If its a testing concern, shouldn't it be on a top level test fixture to cleanup the environment before or after each test?

Reply:
The issue without this change is that environment that gets used for running the script is the environment without the dotenv variables.  Is there a better way to accomplish it?

Reply:
> shouldn't it be on a top level test fixture to cleanup the environment before or after each test?

Oh yeah, it is cleaned up.